curious/urls.py

```diff
@@ -7,7 +7,7 @@ def redirect_to_static(request):
   path = request.get_full_path()
   return HttpResponseRedirect('/static%s' % path)
 
-urlpatterns = patterns('',
+urlpatterns = [
```

Comment:
why no longer a tuple?

Reply:
`patterns` no longer exists and I alter urlpatterns enough that it's more of a convenience to keep them mutable.